### PR TITLE
Earth 1405 blazy background images

### PR DESCRIPTION
**Pull request by aaroncole**
# NOT READY

# Summary
- Update all P/Ts with CSS background images to use lazy-loading

# Needed By (Date)
- Next production push

# Steps to Test

1. Open any page with a Section Expandable, Section Highlight Banner,  Postcard, and/or Link Banner.
2. Also, check the global footer image.
3. Open the Network panel in Devtools (Chrome/Firefox)
4. Watch to see if the images load only once their content is scrolled into the viewport

# Affected versions or modules
- None

# Associated Issues and/or People
- EARTH-1505

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by buttonwillowsix** [comment]
@aaroncole is this PR ready? Just realized it says "Not Ready."

File: templates/components/global-footer/global-footer.html.twig [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #}
 
 <footer id="footer__container">
-  <section class="contact-footer" style="background-image:url('{{ base_path ~ directory }}/img/optimized/footer-bg.jpg')";>
+  <section class="contact-footer b-lazy bg" data-src="{{ base_path ~ directory }}/img/optimized/footer-bg.jpg";>
```
Comment (buttonwillowsix) [inline templates/components/global-footer/global-footer.html.twig:9]: @aaroncole when I check out this branch, I am still seeing 

`<section class="contact-footer b-lazy bg b-loaded" ;="" style="background-image: url(&quot;/themes/stanford/matson/img/optimized/footer-bg.jpg&quot;);"><div class="contact-footer__brand"><div id="footer__block-matson-branding" class="system-branding-block block--lockup block-grid-collapse block-system footer__block-matson-branding"><div class="block--lockup__site-logo-wrapper"><a href="/" title="Stanford Earth | School of Earth, Energy &amp; Environmental Sciences" rel="home" class="block--lockup__site-logo">Stanford Earth | School of Earth, Energy &amp; Environmental Sciences</a></div><div class="block--lockup__site-name-and-slogan">
        Stanford Earth
      </div><div class="block--lockup__site-slogan">School of Earth, Energy &amp; Environmental Sciences</div></div></div><div class="contact-footer__social"><h2>Join us on Social Media</h2><ul class="social-links"><li><a href="https://twitter.com/stanfordearth"><span class="visually-hidden">Twitter</span><svg class="i-svg i-svg--twitter is-white" role="img" aria-labelledby="svg_twitter"><title id="svg_twitter">Twitter</title><use xlink:href="#i-twitter" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></li><li><a href="https://www.facebook.com/stanfordearth"><span class="visually-hidden">Facebook</span><svg class="i-svg i-svg--fb is-white" role="img" aria-labelledby="svg_facebook"><title id="svg_facebook">Facebook</title><use xlink:href="#i-fb" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></li><li><a href="https://www.instagram.com/stanfordearth/"><span class="visually-hidden">Instagram</span><svg class="i-svg i-svg--instagram is-white" role="img" aria-labelledby="svg_instagram"><title id="svg_instagram">Instagram</title><use xlink:href="#i-instagram" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></li><li><a href="https://www.linkedin.com/company/stanford-earth/"><span class="visually-hidden">Linkedin</span><svg class="i-svg i-svg--linkedin is-white" role="img" aria-labelledby="svg_linkedin"><title id="svg_linkedin">Linkedin</title><use xlink:href="#i-linkedin" xmlns:xlink="http://www.w3.org/1999/xlink"></use></svg></a></li></ul></div><div class="contact-footer__links"><ul class="menu"><li class="menu-item"><a href="/alumni" title="Alumni Page" data-drupal-link-system-path="node/3316">Alumni</a></li><li class="menu-item"><a href="/giving" title="Give to Stanford Earth" data-drupal-link-system-path="node/3886">Give to Stanford Earth</a></li><li class="menu-item"><a href="/news-events/media-mentions" title="Newsroom" data-drupal-link-system-path="node/2086">Newsroom</a></li><li class="menu-item"><a href="/about/contact-us" title="Contact Form" data-drupal-link-system-path="node/2246">Contact</a></li><li class="menu-item"><a href="https://stanford.service-now.com/it_services?id=sc_cat_item&amp;sys_id=676f7ee513286b40d3b6b3b12244b0e5" title="Link to Website Feedback form in ServiceNow">Provide Website Feedback</a></li><li class="menu-item"><a href="https://soap.stanford.edu/contact-us/report-web-accessibility-issue">Report accessibility issues</a></li><li class="menu-item"><a href="/saml_login" data-drupal-link-system-path="saml_login">Login</a></li></ul></div><div class="contact-footer__subscribe"><h2>Subscribe to Stanford Earth Matters</h2><p>Our Monthly Research News Alert</p><div id="mc_embed_signup"><form action="//stanford.us7.list-manage.com/subscribe/post?u=167e9da7acec83c4cc802b3e7&amp;id=69d6fa9e3a" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate subscribe-form" target="_blank" novalidate=""><div id="mc_embed_signup_scroll"><input type="email" value="" name="EMAIL" class="email email-input" id="mce-EMAIL" placeholder="name@stanford.edu" required="" aria-label="Email Address"><div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_167e9da7acec83c4cc802b3e7_69d6fa9e3a" tabindex="-1" value=""></div><div class="clear"><input type="submit" value="Subscribe" name="Join" id="mc-embedded-subscribe" class="button button__hollow" aria-label="Subscribe"></div></div></form></div></div></section>`

Comment (buttonwillowsix) [inline templates/components/global-footer/global-footer.html.twig:9]: Hm, on my local on this branch I see this change in code, but don't see it reflected on my site: 
'<section class="contact-footer b-lazy bg b-loaded" ;="" data-bio-hit="1" style="background-image: url(&quot;/themes/stanford/matson/img/optimized/footer-bg.jpg&quot;);"></section>'

Comment (buttonwillowsix) [reply templates/components/global-footer/global-footer.html.twig:9]: Although I _do_ see the classes. Weird.

File: patterns/molecules/postcard/postcard.html.twig [hunk 2]
```diff
@@ -7,7 +7,7 @@
 {% set attributes = attributes.addClass(['postcard', variants.image_orientation, variants._text_orientation]) %}
 
 <div{{ attributes }}> {# has-image-right, has-text-offset, has-text-inset #}
-  <div class="postcard__image" style="background-image:url('{{ image|get_img_src }}')"></div>
+  <div class="postcard__image b-lazy bg" data-src="{{ image|get_img_src }}"></div>
```
Comment (buttonwillowsix) [inline patterns/molecules/postcard/postcard.html.twig:10]: @aaroncole same oddity as the footer. I see the classes you have added, but still see it as 'style='background-image...'

**Review by buttonwillowsix — COMMENTED** [review]
I think we could push this through as is, if you want. The classes are definitely there now. but the method for the images hasn't changed from what I can see. Not sure why!

**Comment by aaroncole** [comment]
I replaced the style attribute with a data-src attribute.

**Comment by buttonwillowsix** [comment]
> I replaced the style attribute with a data-src attribute.

I see that in the commits, but I don't see that in the rendered HTML on my local. Do you?

**Review by buttonwillowsix — APPROVED** [review]

**Pull request merged**